MediaBrowser.Providers/Plugins/MusicBrainz/Configuration/PluginConfiguration.cs

```diff
@@ -0,0 +1,44 @@
+﻿using MediaBrowser.Model.Plugins;
+
+namespace MediaBrowser.Providers.Plugins.MusicBrainz
+{
+    public class PluginConfiguration : BasePluginConfiguration
+    {
+        private string _server = Plugin.Instance.DefaultServer;
```

Comment:
@dkanada DefaultServer is a const, you should access with `Plugin.DefaultServer` not via `Instance`. Same with the references to `DefaultRateLimit`

> 2020-02-29T07:26:39.2721421Z Plugins/MusicBrainz/Configuration/PluginConfiguration.cs(7,34): error CS0176: Member 'Plugin.DefaultServer' cannot be accessed with an instance reference; qualify it with a type name instead [/home/vsts/work/1/s/MediaBrowser.Providers/MediaBrowser.Providers.csproj]